.travis.yml

```diff
@@ -0,0 +1,23 @@
+os: linux
+services: docker
+language: minimal
+install: skip
+addons:
+  apt:
+    packages:
+    - docker-ce
+script:
+  - USE_DOCKER="true" ./build/build_all.sh
+  - "./build/build_img.sh"
+deploy:
+  provider: script
+  skip_cleanup: true
+  script:
+    - echo "$DOCKER_PASS" | docker login -u "$DOCKER_USER" --password-stdin && docker push filebrowser/filebrowser && docker logout
+  on:
+    repo: 1138-4EB/filebrowser
+    branch: master
+env:
+  global:
+  - secure: YtxLrM3z2A/wDh5yLXrMXgsMRr+c7v5auS/2SpNxra0E6fcFKNPtk2g6JB4JfpFRp6feeuwtT61b16OdHjNVcwtZPLEdM1o7nUaLd+UxxKYlXwMSl+Hqaclx5KbBwlVCU3BKJjPVY/RujjSgOGG7C9b40E8YBssVIxyM51qV+u4C7Odu99ZC42OnFG8n+SsdP2xdI7BFw5qJTmNAgnns4QEAiZi++YMtsYmoo828mNpYX093N1VeW81JHeHo82Xwifpeij2fxuOkGAf2JMOuYrH+Mn+jgAiSUGkNRKhaA5do3ov6tSiCWZtQjrAIFk56knXaeeIO+ysryJ+BO9w2qquI1IYktpIemZjjoz24qKeUY0suxPdEe7cH17/P6n2LHUyXiUyUpjRX2bFqo6kfAoMc6oPqgOWu8Ls5GwuEHqCwsZXTVy4ZqNYiAIgk85lUecm9uy+Ubtu4bCdJpDlkC9jGow4Wn/6lHNxsuGWfBR1P94WtC+qmrZf/xje3A5xKATtZtLrfc0tLdsH5VYDyIZ4WYWOIrCemayjXK3RJxN+qQDxaPWPG0cmHAxPslUyHE1eDxy1hCkCZx82Qjr1dX8Xx5tO4SoBayV9pmJo4+IyEb711NQyYrA/EdDswMmrVJNLBY+yDK6umkMlcIuRkVQanOydgRrusbKuQoAGOQRE=
```

Comment:
@1138-4EB just a bit confused: what's this?

Reply:
In order to push docker images from travis-ci to hub.docker.com, docker credentials need to be set. These are added as encrypted variables through the travis CLI (a ruby gem you use locally). Those two `secure` strings are the hashes corresponding to envvars `$DOCKER_PASS` and $DOCKER_USER`. When a build is executed, travis decrypts those and "ensures" that are not shown in the logs.

For further info see (in this order):

- https://docs.travis-ci.com/user/docker/#Pushing-a-Docker-Image-to-a-Registry
- https://docs.travis-ci.com/user/environment-variables#Encrypting-environment-variables
- https://docs.travis-ci.com/user/build-stages/share-docker-image/

Reply:
It is to be noted that those specific hashes correspond to my repo and the docker user I am using. Therefore, when this PR is merged, two new hashes should be created by i) a GitHub user that has write permissions to write in this repo and ii) using a Docker user that has write permissions to hub.docker.com/u/filebrowser.The same person needs to know both credentials, in order to avoid sharing passwords between different people.